### ISSUE TITLE
2.5.2 / 2.4.6 patch releases plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#2057]
- #2034 
- #2038 
- #2040 
- #2048
- #2046 
- #2066
- #2063

**Comment by joeparsons** [comment]
## Pick list

```
git cherry-pick cd9bd00c09ff5aae8c60587ee5fbb2f6876aa745
git cherry-pick d72239c067c70cf5f4e30645ee6be5d320bc3762
git cherry-pick bdbe9ff8983725ca51c2ea1d559dbd278bf19a7a
git cherry-pick 7b5421fab5d50c7c0c7aef996556ca7437f7e9d5
git cherry-pick 2ee18256aef764f7370d46ef42a8ce98d3406881
git cherry-pick 3d6003b59f920cebcc4ca8ccef823ce3e80f99db
git cherry-pick 45704499f507ff185b6fe3e03977abd813abb017
```

**Comment by joeparsons** [comment]
## Release notes snippet

```
- Closes #2062: Skip migrate processing for event link field values if missing. (#2063)
- Closes #2065 Database connection target and key are swapped. (#2066)
- Closes #2045: Add patch for Drupal core PHP 8.1 deprecation errors affecting status report. (#2046)
- Update drupal/media_library_edit requirement from 2.3.0 to 3.0.1 (#2048)
- Closes #1695: Replaced drush en with drush install (#2040)
- Closes #2035 - Include notice that "reordering overrides alphabetical sorting" (#2038)
- Closes #2031 - Fix unusable row weights in reorder view (#2034)
- Back to dev.
```

**Comment by joeparsons** [comment]
Releases created:
https://github.com/az-digital/az_quickstart/releases/tag/2.4.6
https://github.com/az-digital/az_quickstart/releases/tag/2.5.2
